README.md

```diff
@@ -27,14 +27,6 @@
   </a>
 </p>
 
-<p align="center">
-  <strong>
-    <a href="docs/faq.md">FAQ</a>
-    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
-    <a href="SECURITY.md">Security</a>
```

Comment:
I think we should leave the security link here

Reply:
Restored to the top (three links in the first row)